test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -2824,6 +2827,52 @@ TEST_F(ClusterManagerInitHelperTest, UpdateAlreadyInitialized) {
   cluster2.initialize_callback_();
 }
 
+TEST_F(ClusterManagerInitHelperTest, InitSecondaryWithoutEdsPaused) {
+  InSequence s;
+
+  ReadyWatcher cm_initialized;
+  init_helper_.setInitializedCb([&]() -> void { cm_initialized.ready(); });
+
+  NiceMock<MockClusterMockPrioritySet> cluster1;
+  ON_CALL(cluster1, initializePhase()).WillByDefault(Return(Cluster::InitializePhase::Secondary));
+  init_helper_.addCluster(cluster1);
+
+  const auto& type_url = Config::TypeUrl::get().ClusterLoadAssignment;
+  EXPECT_CALL(cm_.ads_mux_, paused(Eq(ByRef(type_url)))).Times(1).WillRepeatedly(Return(false));
+  EXPECT_CALL(cm_.ads_mux_, pause(Eq(ByRef(type_url)))).Times(1);
```

Comment:
Nit: `Times(1)` is redundant everywhere.

Reply:
just want to be explicitly here :)

Reply:
Yeah, but it doesn't really say anything new, by default `EXPECT_CALL` is `Times(1)`. I'm thinking of writing a `check_format.py` script to ban use of this in the repo, so I recommend removing it here :)

Reply:
removed ;)